src/Files.App/Dialogs/BulkRenameDialog.xaml.cs

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2024 Files Community
+// Licensed under the MIT License. See the LICENSE.
+
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+
+namespace Files.App.Dialogs
+{
+	public sealed partial class BulkRenameDialog : ContentDialog, IDialog<BulkRenameDialogViewModel>
+	{
+		private FrameworkElement RootAppElement
+			=> (FrameworkElement)MainWindow.Instance.Content;
+
+		public BulkRenameDialogViewModel ViewModel
```

Comment:
This has been a bad practice and I was going to delete.
Can you use binding instead?

Reply:
I don't think that works for `IDialog`.

Reply:
Omg. Okay, let's leave as is for now.